src/components/Image/Avatar.js

```diff
@@ -10,6 +10,8 @@ const Wrapper = styled(Box)`
   ${variant({
     prop: 'size',
     variants: {
+      xsmall: { width: '12px', height: '12px' },
+      smaller: { width: '16px', height: '16px' },
```

Comment:
this is the first time anywhere where we have `small`, `smaller` and `xsmall`.
As in `smaller` does't exist anywhere else.

Do we need so many different sizes? I kinda like to keep scales as consistent as possible across different components.

Reply:
Will remove